ntoskrnl/mm/pagefile.c

```diff
@@ -329,8 +330,9 @@ MmAllocSwapPage(VOID)
             }
             MiUsedSwapPages++;
             MiFreeSwapPages--;
-            KeReleaseGuardedMutex(&MmPageFileCreationLock);
+            UpdateTotalCommittedPages(1);
```

Comment:
As I think I said somewhere before, this can be called outside of the page file creation lock. But NBD either way.

Reply:
Would you like me to swap both lines or is it ok for approval?

Reply:
I'm personally fine with updating all counters at once (instead of having some counters with the lock held, then the other counter(s) with the lock unheld).

Reply:
So I keep it as is since it's ok for @ThFabba too.

I let you answer on the comment and we're done !